### ISSUE TITLE
Changes to Make

### ISSUE THREAD

**Issue by cleemullins** [thegreatsantini/DejaFood-bot#3]
When you have a chance, please do the following:
1. Get the bot integrated with WebChat, and running on Azure. Send met he resulting URL. 
2. Cleanup all the keys, so GitHub doesn't have any secretes checked in. 

When this is done, ping me by adding to this issue.

**Comment by thegreatsantini** [comment]
I removed the .bot files with the secrets and keys. I am working on deploying it using

```msbot clone services --folder deploymentScripts/msbotClone --location "westus2" --verbose --code-dir . luisAuthoringKey <AUTHKEY>  --name "DeployedBot"--subscriptionKey <sub_KEY>```

I was get an error module not found `Azure.CLI.telemetry` but I think i was able to bypass this issue. 
The CLI gave me a decrypt secret for the newly created bot. However, now the bot doesn't work locally any more, an error with LUIS config. I am following this link https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-deploy-az-cli?view=azure-bot-service-4.0&tabs=js. The article was written 4 days ago. I might be running into some bugs that they don't know about yet.

**Comment by cleemullins** [comment]
Thanks for the update. We may have some bugs in that process, and this is the type of things you would be:
1. Helping people with
2. Fixing. :) 

//Chris

**Comment by cleemullins** [comment]
How is it going?

**Comment by thegreatsantini** [comment]
Just got back. Had an in person and phone interview.

**Comment by thegreatsantini** [comment]
Still not sure why even my local isn't working. After I deployed the bot using the `msbot clone services` script my LUIS config must have changed or something.  If I had the time,I would nuke the  resources and create a bot through the azure portal.  I will do it regardless, I want to have it deployed for my portfolio. Also my friend has a meal prepping web app that would be fun to have this bot on there.

**Comment by cleemullins** [comment]
Ok, let me know when/if you get this working. I would suggest you post a question on StackOverflow, tag it with "BotFramework" and see if they can help. :)

**Comment by thegreatsantini** [comment]
Good idea. I will work on this weekend. I have interviews and take homes the rest of the week. Also did you get a chance to read my write up in this repo?
